_posts/2022-06-26-news.md

```
@@ -0,0 +1,10 @@
+---
+title: News
+category: Jekyll
+layout: post
+---
+
+_Stay Updated_
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)